test/src/main/java/org/apache/accumulo/test/functional/HalfDeadServerWatcherIT.java

```diff
@@ -155,7 +161,14 @@ public void testTwo() throws Exception {
       assertTrue(testTabletServerWithStuckWatcherDies());
     } else {
       // This test should time out
-      assertThrows(IllegalStateException.class, () -> testTabletServerWithStuckWatcherDies());
+      try {
+        testTabletServerWithStuckWatcherDies();
+        fail("Test did not time out.");
+      } catch (IllegalStateException e) {
+        if (!e.getMessage().contains("Timeout exceeded")) {
+          fail("Unexpected exception: " + e.getMessage());
+        }
```

Comment:
The original assertThrows was better. assertThrows returns the thrown exception, so you can follow it with an asssertTrue for checking the message.

Reply:
Reverted change and implemented suggestion in a3ce01d. I forgot that assertThrows returns the Exception. Thanks for the reminder!